### ISSUE TITLE
log format of ACK controllers

### ISSUE THREAD

**Issue by bin3377** [aws-controllers-k8s/community#1764]
**Describe the bug**

I installed the iam-controller with helm chart and I'm trying to find a way to get json log from the container to monitor the iam changes. but seems no matter which value I set for `ACK_ENABLE_DEVELOPMENT_LOGGING` or `ACK_LOG_LEVEL`, the log still shows as:

```
2023-04-06T20:56:02.935Z        INFO    ackrt   desired resource state has changed      {json}
```

which is a combination of plain string and json, extremely hard to parse

**Steps to reproduce**
install `iam-chart` from `oci://public.ecr.aws/aws-controllers-k8s` with version `v1.1.2` with all default values

**Expected outcome**
a switch to enable pure json log

**Environment**
iam chart version v1.1.2

* Kubernetes version
1.22

* Using EKS (yes/no), if so version? 
yes, `v1.22.17-eks-48e63af`

* AWS service targeted (S3, RDS, etc.)
IAM

**Comment by jaypipes** [comment]
Hi @bin3377! I removed the IAM part of the issue summary since all ACK controllers have the same logging infrastructure and format. This logging infrastructure is the same as all binaries that use the standard upstream kubernetes controller-runtime logging setup (which [all ACK controllers do](https://github.com/aws-controllers-k8s/runtime/blob/e526d548a7524353604b8b20c0511714ac2a86d8/pkg/config/config.go#L177-L188)). The log format is always `{TIMESTAMPZ}\t{LOG_LEVEL_STRING}\t{LOG_MESSAGE}\t{STRUCTURED_LOG_TAGS}`

The `{STRUCTURED_LOG_TAGS}` is always a JSON object containing a set of key/value pairs that provide more detailed context information about the resources being operated on during that log line. Common structured log keys are "kind", where the corresponding value is the Kind of the resource, "name" where the corresponding value is the name of the resource, etc.
